src/memory_targets/memory_target_numa.c

```diff
@@ -176,23 +174,72 @@ static umf_result_t numa_get_bandwidth(void *srcMemoryTarget,
 
     // Given NUMA nodes aren't local, HWLOC returns an error in such case.
     if (!hwloc_bitmap_intersects(srcNumaNode->cpuset, dstNumaNode->cpuset)) {
-        *bandwidth = 0;
+        *value = 0;
         return UMF_RESULT_SUCCESS;
     }
 
+    enum hwloc_memattr_id_e hwlocMemAttrType = INT_MAX;
+    switch (type) {
+    case MEMATTR_TYPE_BANDWIDTH:
+        hwlocMemAttrType = HWLOC_MEMATTR_ID_BANDWIDTH;
+        break;
+    case MEMATTR_TYPE_LATENCY:
+        hwlocMemAttrType = HWLOC_MEMATTR_ID_LATENCY;
+        break;
+    default:
+        assert(0); // Shouldn't be reachable.
```

Comment:
please return invalid arg in default case - note that in the release code this assert will do nothing

Reply:
Done.